solidity/contracts/converter/BancorConverter.sol

```diff
@@ -507,8 +507,13 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
         validReserve(_reserveToken)
         returns (uint256, uint256)
     {
+        uint256 supply = token.totalSupply();
+
+        if (supply == 0)
+            return (_depositAmount, _depositAmount);

```

Comment:
I'm pretty sure the 2nd value here should be 0 (fee), no?

Reply:
@yudilevi : No, it should be amount minus fee.

Reply:
@yudilevi : Damn, you're right!

Reply:
@yudilevi : Fixed.